cedar-policy-core/src/parser/cst_to_ast/to_ref_or_refs.rs

```diff
@@ -253,6 +313,69 @@ impl Node<Option<cst::Primary>> {
                 .into()),
         }
     }
+
+    #[cfg(feature = "tolerant-ast")]
+    fn to_ref_or_refs_tolerant_ast<T: RefKind>(&self, var: ast::Var) -> Result<T> {
```

Comment:
Same comment as earlier. I can't easily spot the difference between this implementation and the other one.

Reply:
Will refactor